### PR TITLE
Urgent note in doc per customer issue re: LDAP config

### PR DESCRIPTION
**Pull request by sfortner-RH**
Applicable to dedicated docs for configuring LDAP.

### CODE REVIEW

File: modules/identity-provider-about-ldap.adoc [hunk 1]
```diff
@@ -52,6 +52,11 @@ If the scope is not provided, the default is to use a scope of `sub`.
 `(objectClass=*)`
 |===
 
+[IMPORTANT]
+====
+If you're using an insecure LDAP connection (ldap:// or port 389), then you must check the *Insecure* option in the configuration wizard.
```
Comment (bobfuru) [inline modules/identity-provider-about-ldap.adoc:57]: s/you're/you are/
avoid contractions

**Review by bobfuru — COMMENTED** [review]
one small nit, otherwise lgtm

File: modules/identity-provider-about-ldap.adoc (hunk 1 above)
Comment (sfortner-RH) [reply modules/identity-provider-about-ldap.adoc]: Fixed- Ready to merge

**Comment by sfortner-RH** [comment]
@wgordon17 FYI :)

**Pull request merged**